commit afeb10079418b5bb1a2e31b7aa60b83bc6d51750 (refs/changes/92/192/1)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2016-08-02T12:56:17-07:00 (3 years, 2 months ago)
    
    HEAD-2319 Remove vestigal datacenter_headnode_id in answers.json examples
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/answers.json.tmpl b/answers.json.tmpl
index 033b9044..7656e1af 100644
--- a/answers.json.tmpl
+++ b/answers.json.tmpl
@@ -8,7 +8,6 @@
     "region_name": "coal",
     "datacenter_name": "coal",
     "datacenter_location": "My Laptop",
-    "datacenter_headnode_id": "<default>",
     "admin_nic": "00:50:56:34:60:4c",
     "admin_ip": "10.99.99.7",
     "admin_provisionable_start": "<default>",
diff --git a/answers.json.tmpl.external b/answers.json.tmpl.external
index da72b032..38f495c1 100644
--- a/answers.json.tmpl.external
+++ b/answers.json.tmpl.external
@@ -8,7 +8,6 @@
     "region_name": "coal",
     "datacenter_name": "coal",
     "datacenter_location": "My Laptop",
-    "datacenter_headnode_id": "<default>",
     "admin_nic": "00:50:56:34:60:4c",
     "admin_ip": "10.99.99.7",
     "admin_provisionable_start": "<default>",
